installer/templates/new/widgets/text/text.scss

```diff
@@ -19,14 +19,20 @@ $moreinfo-color:    rgba(255, 255, 255, 0.7);
 
   .more-info {
     color: $moreinfo-color;
+    font-size: 24px;
   }
 
   .updated-at {
     color: rgba(255, 255, 255, 0.7);
   }
 
 
-  &.large h3 {
+  & h3 {
     font-size: 65px;
   }
 }
+
+.widget-text-custom {
+
+    background-color: #ff0000;
+}
```

Comment:
this is where I defined my custom css for the `specialStyle` attribute.

I wonder if we shouldn't do an include statement in the widget scss to pick up custom (user-defined) styles instead?